### ISSUE TITLE
[Tarea 1] Actividad 4: ¿Qué métricas evaluar?

### ISSUE THREAD

**Issue by SergioGazali** [PUC-RecSys-Class/RecSysPUC-2021#13]
Hola!
Al principio del enunciado se dice que debemos evaluar para cada modelo ndcg@10, ndcg@20, ndcg@30, map@10, map@20, map@30. Sin embargo la actividad 4 pide un análisis de sensibilidad sólo para ndcg@10 y map@10. Lo que interpreto es que las métricas mencionadas al principio (@10, @20, @30) son las que indicaremos en la actividad 5. Es decir, que en la actividad 4 debemos seleccionar una combinación de hiperparámetros sólo en base a las métricas @10, para luego obtener el desempeño de esos hiperparámetros en métricas @10, @20, @30 en la actividad 5. ¿Esto es correcto?

Muchas gracias!

**Comment by alfa-labarca** [comment]
Correcto, como mencioné en #3, para el análisis de sensibilidad se utiliza solamente ndcg@10 y map@10, pero en cualquier otra instancia en que se pida evaluar un modelo sin especificar las métricas, se deben usar todas las métricas pedidas en el comienzo del enunciado.

Saludos!
